CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Always place a blank line after script tags (#782).
 * Don't add unneeded splits on if elements near comments (#888).
 * Indent blocks in initializers of multiple-variable declarations.
+* Update the null-aware subscript syntax from `?.[]` to `?[]`.
```

Comment:
tis nice to tell folks you bumped the min SDK here, too! 😊

Reply:
This doesn't seem CHANGELOG worthy to me.